### PR TITLE
Cleaning up DropOffUtil.getFileType

### PR DESCRIPTION
**Pull request by fbruton**

### CODE REVIEW

File: src/main/java/emissary/output/DropOffUtil.java [hunk 1]
```diff
@@ -772,6 +771,13 @@ public String getFileType(final Map<String, String> metaData, final String forms
         return fileType;
     }
 
+    /**
+     * Parameters that are used to determine filetype in {@link #getFileType(Map, String)}
+     */
+    public enum FileTypeCheckParameter {
+        POPPED_FORMS, COMPLETE_FILETYPE, FILETYPE, FontEncoding, FINAL_ID,
```
Comment (ets-mlb) [inline src/main/java/emissary/output/DropOffUtil.java]: Use uppercase and create constructor to set the actual values.

File: src/main/java/emissary/output/DropOffUtil.java [hunk 2]
```diff
@@ -772,6 +771,24 @@ public String getFileType(final Map<String, String> metaData, final String forms
         return fileType;
     }
 
+    /**
+     * Parameters that are used to determine filetype in {@link #getFileType(Map, String)}
+     */
+    public enum FileTypeCheckParameter {
+        COMPLETE_FILETYPE("COMPLETE_FILETYPE"), FILE_TYPE("FILETYPE"), FINAL_ID("FontEncoding"), FONT_ENCODING("FontEncoding"), POPPED_FORMS(
```
Comment (ets-mlb) [inline src/main/java/emissary/output/DropOffUtil.java]: FINAL_ID("FontEncoding") -> FINAL_ID("FINAL_ID")

File: src/main/java/emissary/output/DropOffUtil.java [hunk 3]
```diff
@@ -729,39 +729,38 @@ public String getFileType(final Map<String, String> metaData) {
...
             }
-            if (fileType == null || fileType.length() == 0) {
-                if (metaData.containsKey("FontEncoding")) {
+            if (fileType.length() == 0) {
+                if (metaData.containsKey(FileTypeCheckParameter.FONT_ENCODING.getFieldName())) {
                     fileType = "TEXT";
                 }
             }
-            if (fileType == null || fileType.length() == 0) {
+            if (fileType.length() == 0) {
```
Comment (jpdahlke) [inline src/main/java/emissary/output/DropOffUtil.java]: isEmpty?

File: src/main/java/emissary/output/DropOffUtil.java [hunk 4]
```diff
@@ -729,39 +729,38 @@ public String getFileType(final Map<String, String> metaData) {
...
+            if (forms.contains(" ")) {
                 fileType = forms.substring(0, forms.indexOf(" ")).trim();
-                metaData.put("COMPLETE_FILETYPE", forms);
+                metaData.put(FileTypeCheckParameter.COMPLETE_FILETYPE.getFieldName(), forms);
             } else {
                 fileType = forms;
             }
-            if (fileType == null || fileType.length() == 0) {
-                if (metaData.containsKey("FontEncoding")) {
+            if (fileType.length() == 0) {
```
Comment (jpdahlke) [inline src/main/java/emissary/output/DropOffUtil.java]: isEmpty?

**Comment by fbruton** [comment]
> Unit tests?
Fixed

File: src/main/java/emissary/output/DropOffUtil.java (hunk 3 above)
Comment (fbruton) [reply src/main/java/emissary/output/DropOffUtil.java]: isBlank

File: src/main/java/emissary/output/DropOffUtil.java (hunk 2 above)
Comment (fbruton) [reply src/main/java/emissary/output/DropOffUtil.java]: Fixed

File: src/main/java/emissary/output/DropOffUtil.java (hunk 1 above)
Comment (fbruton) [reply src/main/java/emissary/output/DropOffUtil.java]: Fixed

**Pull request merged**